extensions/mssql/package.json

```diff
@@ -400,16 +400,26 @@
               "name": "%title.tasks%",
               "row": 0,
               "col": 0,
-              "colspan": 2,
+              "colspan": 1,
               "widget": {
                 "tasks-widget": [
                   "mssqlCluster.task.newNotebook",
                   "mssqlCluster.task.openNotebook",
-                  "mssqlCluster.livy.task.submitSparkJob",
-                  "mssqlCluster.livy.task.openSparkHistory",
-                  "mssqlCluster.livy.task.openYarnHistory"
+                  "mssqlCluster.livy.task.submitSparkJob"
                 ]
               }
+            },
+            {
+              "name": "%title.endpoints%",
+              "row": 0,
+              "col": 1,
+              "rowspan": 1.5,
```

Comment:
do we support .5 spans? I had thought it was 1 or 2?

Reply:
I tested it and it worked. So I think we're supporting .5 spans.